Tlv9001v2

matyas1995 opened the pull request:
Added the Texas Instruments TLV9001 single op-amp with the SOT-353-5 package.
Took the TLV172 as an example, but centered the symbol.

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

cpresser commented:
Hi and thanks for your contribution.

- [ ] The name needs some rework. You did choose `TLV9001xIDCKR`. I don't see why the wildcard `x` is in there - please enlighten me. The trailing `R` should get removed since it is a packaging option only.
- [x] Pins Numbers and types are okay. The names are not perfect but acceptable (`IN+` instead of just `+`) to keep the library consistent.
- [ ] This should be an alias of `TLV172IDCK`, it does not need to be a new symbol since all the pins and the package are the same.

matyas1995 commented:
Hi,
the wildcard x has been there because I based the part on the TLV9004 that is already in the library. The wildcard is supposed to cover the S variant of the IC that supports shutdown mode. Unfortunately this variant is not entirely compatible and not all packages are supported, so I removed it altogether.
Also, the wildcard in the TLV9004 also does not seem to be of any use, as the shutdown variant does not exist with the RUC package.
I deleted the standalone TLV9001 and recreated it as an alias of TLV172. I will check for possible aliases when creating future symbols.

cpresser commented:
Looks good. 
If you want to add the `S` version, The `LMV601` seems to be a good candidate to add an alias as it comes in `SC-70-6` footprint and has the shutdown pin.
